test/support/case.ex

```diff
@@ -7,11 +7,28 @@ defmodule Sentry.Case do
 
   import Sentry.TestHelpers
 
-  setup do
+  setup context do
     config_before = all_config()
 
     on_exit(fn ->
       assert config_before == all_config()
     end)
+
+    case context[:span_storage] do
+      nil -> :ok
+      true -> setup_span_storage([])
+      opts when is_list(opts) -> setup_span_storage(opts)
+    end
+  end
+
+  defp setup_span_storage(opts) do
+    uid = :rand.uniform(1000000)
```

Comment:
Is this supposed to be unique?

Reply:
@whatyouhide yes

Reply:
Then `:rand.uniform/1` won't guarantee that in any way. Use `System.unique_integer([:positive])` instead.